models.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Date,
+    ForeignKey,
+)
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class DBAuthor(Base):
```

Comment:
Rename it to Atuthor. We never declare DBAuthor to show that is model. In fastapi we use model.Author, schemas.Author etc.